static.ts

```diff
@@ -1,11 +1,13 @@
-import _STATIC from "./static.json";
+import _STATIC from "./static.json" with { type: "json" };
 import { defaultsDeep, get as _get } from "lodash";
 import { ThemeSettings } from "@/theme";
 
 import type { ResultListingComponentOptions } from "@/components/ResultListing";
 import type { ResultComponentOptions } from "@/components/Result";
 import { NavigationOptions } from "@/components/Navigation";
 
+import type { GFacet } from "@globus/sdk/services/search/service/query";
```

Comment:
This isn't going to work based on the current type in the SDK... which I think is slightly off based on:

- `size` should be optional when `type: "terms"`
- `name` is optional and is only required if there is a conflict in `field_name`

Aside from that, it's not clear we're actually processing the generated `GFilter` properly given the configured `GFacet`.